Client/Ecosia/Migration/EcosiaHistory.swift

```diff
@@ -23,6 +23,11 @@ final class EcosiaHistory {
     }
 
     static var start: Date?
+    
+    /// Errors related to the `profile.isShutdown` property.
+    struct DataBaseShutdownError: MaybeErrorType {
```

Comment:
Very minor, but does it make sense to move the struct to the top of the class (above the variables) together with any other structs that might exist for this class?

Reply:
Totally make sense 😄 ,will do it in a separate commit 👌